### PR TITLE
pep8 for axiom and sympy interfaces

### PR DESCRIPTION
**Pull request by fchapoton**
pep8 cleanup for the 2 modified files

### :memo: Checklist

- [x] The title is concise, informative, and self-explanatory.
- [x] The description explains in detail what this PR is about.

### CODE REVIEW

File: src/sage/interfaces/axiom.py [hunk 1]
```diff
@@ -458,14 +457,14 @@ def _eval_line(self, line, reformat=True, allow_use_file=False,
             return out
         # out = out.lstrip()
         i = out.find('\n')
-        out = out[i+1:]
+        out = out[i + 1:]
```
Comment (mkoeppe) [inline src/sage/interfaces/axiom.py]: I think this is incorrect per PEP 8 - https://peps.python.org/pep-0008/#pet-peeves
"in a slice the colon acts like a binary operator, and should have equal amounts on either side (treating it as the operator with the lowest priority)."

**Review by mkoeppe — COMMENTED** [review]
Otherwise LGTM

File: src/sage/interfaces/axiom.py (hunk 1 above)
Comment (kwankyu) [reply src/sage/interfaces/axiom.py]: There is equal amounts of space on either side of `:`, namely, zero. I see no problem here.

Comment (kwankyu) [reply src/sage/interfaces/axiom.py]: Ah, you meant
```
# Correct:
ham[lower+offset : upper+offset]
# Wrong:
ham[lower + offset:upper + offset]
```
You are right. 

But personally I cannot agree with the recommendation. I like the "wrong" format :-)

**Review by mkoeppe — APPROVED** [review]

**Pull request merged**